CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
     - [Other notable changes](#other-notable-changes)
 - [v1.3.0-alpha.3](#v130-alpha3)
   - [Downloads](#downloads)
+- [v1.3.0-alpha.3](#v130-alpha3)
```

Comment:
There's a bug in hack/update-generated-docs.sh that creates this line that we don't need. Remember to remove it manually before merging.